src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,36 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final Object[] KEYS = new Object[DEFAULT_CAPACITY];
+    private static final Object[] VALUES = new Object[DEFAULT_CAPACITY];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is the default value